cmd/nerdctl/image_inspect.go

```diff
@@ -95,11 +102,32 @@ func ImageInspectAction(clicontext *cli.Context) error {
 		}
 	}
 
-	b, err := json.MarshalIndent(f.entries, "", "    ")
-	if err != nil {
-		return err
+	var tmpl *template.Template
+	switch format := clicontext.String("format"); format {
+	case "":
+		b, err := json.MarshalIndent(f.entries, "", "    ")
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(clicontext.App.Writer, string(b))
+	case "raw", "table":
+		return errors.New("unsupported format: \"raw\" and \"table\"")
+	default:
+		var err error
+		tmpl, err = templates.Parse(format)
```

Comment:
tmpl is unused

Reply:
hi, actually, I uesd this value(tmp1) in line 121 to check wether the value of tmp1 equal nil or not.

Reply:
i know it